driver/src/price_finding/price_finder_interface.rs

```diff
@@ -20,34 +22,34 @@ impl Default for Fee {
 }
 
 #[derive(Clone, Debug, Copy, PartialEq)]
-pub enum SolverConfig {
+pub enum SolverType {
     NaiveSolver,
-    StandardSolver { solver_time_limit: u32 },
-    FallbackSolver { solver_time_limit: u32 },
+    StandardSolver,
+    FallbackSolver,
 }
 
-impl SolverConfig {
-    pub fn new(solver_type_str: &str, solver_time_limit: u32) -> Result<Self> {
+impl FromStr for SolverType {
+    type Err = Error;
+
+    fn from_str(solver_type_str: &str) -> Result<Self> {
         match solver_type_str.to_lowercase().as_str() {
-            "standard-solver" => Ok(SolverConfig::StandardSolver { solver_time_limit }),
-            "fallback-solver" => Ok(SolverConfig::FallbackSolver { solver_time_limit }),
-            "naive-solver" => Ok(SolverConfig::NaiveSolver),
+            "standard-solver" => Ok(SolverType::StandardSolver),
+            "fallback-solver" => Ok(SolverType::FallbackSolver),
+            "naive-solver" => Ok(SolverType::NaiveSolver),
             _ => Err(anyhow!("solver type does not exit")),
         }
     }
 }
 
-impl SolverConfig {
+impl SolverType {
     pub fn to_args(self) -> Vec<String> {
         match self {
-            SolverConfig::StandardSolver { solver_time_limit } => {
-                vec![format!("--solverTimeLimit={:}", solver_time_limit)]
-            }
-            SolverConfig::FallbackSolver { solver_time_limit } => vec![
-                format!("--solverTimeLimit={:}", solver_time_limit),
+            SolverType::StandardSolver => vec![],
+            SolverType::FallbackSolver => vec![
+                String::from("--tokenInfo=/app/batchauctions/scripts/token_info_mainnet.json"),
```

Comment:
I am curious about this line. It wasn't deleted anywhere, so it is new. Is this intended?

Reply:
Good catch! That was a rebase error that snuck in.